resources/views/service/edit_registration.blade.php

```diff
@@ -108,20 +111,47 @@
                     </ul>
                 </div>
                 <div class="warning">
-                    @foreach( $family->getNoticeReasons() as $notices)
+                    @foreach( $family->getNoticeReasons() as $notices )
                         <p><i class="fa fa-exclamation-circle" aria-hidden="true"></i>
                             Warning: {{ $notices['count'] }} {{ str_plural($notices['entity'], $notices['count']) }}
                             currently "{{ $notices['reason'] }}"</p>
                     @endforeach
                 </div>
-                <div class="attention">
-                    <p><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Reminder: Have you sent the food diary and pie chart yet?</p>
-                </div>
                 <div class="print-button">
                     <button onclick="window.open( '{{ URL::route( "service.registration.print", ["id" => $registration->id]) }}' ); return false">
                         Print a 4 week collection sheet for this family
                     </button>
                 </div>
+                @if ( count($registration->getReminderReasons()) > 0 )
+                    <div class="attention">
+                        <h2>Reminders:</h2>
+                        @foreach ( $registration->getReminderReasons() as $reminder )
+                            <p><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                                {{ $reminder['entity'] }} has {{ $reminder['reason'] }}</p>
+                        @endforeach
+                        @if ( (Auth::user()->can('updateChart', App\Registration::class)) || (Auth::user()->can('updateDiary', App\Registration::class)) )
+                            <div>
+                                <h2>Documents Received:</h2>
+                                @can('updateChart', App\Registration::class)
```

Comment:
Should you use the $registration var as below or this Class?

Reply:
Hmm. not sure; weirdly, both work. I'll make it the class,